cmd/sfncli/runner.go

```diff
@@ -52,20 +53,22 @@ func (t TaskRunner) handleProcessError(ctx context.Context, title string, err er
 // environment and command line params
 func (t TaskRunner) Process(ctx context.Context, args []string, input string) error {
 	if t.sfnapi == nil { // if New failed :-/
-		return t.handleProcessError(ctx, "process-init", fmt.Errorf("NewTaskFailure -- nil sfnapi"))
+		return t.handleProcessError(
+			ctx, "process-init", "unknown", fmt.Errorf("NewTaskFailure -- nil sfnapi"),
+		)
 	}
 
 	var taskInput map[string]interface{}
 	if err := json.Unmarshal([]byte(input), &taskInput); err != nil {
 		return t.handleProcessError(
-			ctx, "process-input", fmt.Errorf("Input must be a json object: %s", err),
+			ctx, "process-input", "unknown", fmt.Errorf("Input must be a json object: %s", err),
 		)
 	}
 	executionName, _ := taskInput["_EXECUTION_NAME"].(string)
 
 	marshaledInput, err := json.Marshal(taskInput)
 	if err != nil {
-		return t.handleProcessError(ctx, "process-input", err)
+		return t.handleProcessError(ctx, "process-input", executionName, err)
```

Comment:
do we have an execution_id or workflow_id here so that we are consistent with what we pass to the worker itself?

Reply:
derp see that you already mentioned this in your PR description. Why not do that directly?